diff --git a/.env b/.env
index 0fd2523..a9a3ac1 100644
--- a/.env
+++ b/.env
@@ -1 +1 @@
-VERSION=latest
+VERSION=1.2.2
diff --git a/Dockerfile b/Dockerfile
index 3a7c2fc..ffa967d 100644
--- a/Dockerfile
+++ b/Dockerfile
@@ -16,7 +16,7 @@ RUN apk update && \
   rm -rf \
     /var/cache/apk/*
 
-ENV KANBOARD_VERSION 1.0.43
+ENV KANBOARD_VERSION 1.2.2
 ENV KANBOARD_TARBALL https://github.com/kanboard/kanboard/archive/v${KANBOARD_VERSION}.tar.gz
 
 RUN curl -sLo - \
diff --git a/README.md b/README.md
index 0eee53e..5bc045f 100644
--- a/README.md
+++ b/README.md
@@ -15,7 +15,6 @@ These are docker images for [Kanboard](https://kanboard.net/) running on an [PHP
 ## Volumes
 
 * /var/lib/kanboard
-* /srv/www/vendor
 
 
 ## Ports
@@ -27,7 +26,7 @@ These are docker images for [Kanboard](https://kanboard.net/) running on an [PHP
 
 ```bash
 ENV KANBOARD_DEBUG false
-ENV KANBOARD_PLUGIN_API_URL https://kanboard.net/plugins.json
+ENV KANBOARD_PLUGIN_API_URL https://kanboard.org/plugins.json
 ENV KANBOARD_PLUGIN_INSTALLER true
 ENV KANBOARD_CACHE_DRIVER memory
 ENV KANBOARD_MAIL_CONFIGURATION true
@@ -96,6 +95,7 @@ ENV KANBOARD_HTTP_PROXY_USERNAME
 ENV KANBOARD_HTTP_PROXY_PASSWORD
 ENV KANBOARD_HTTP_VERIFY_SSL_CERTIFICATE true
 ENV KANBOARD_TOTP_ISSUER Kanboard
+ENV KANBOARD_EXTERNAL_AUTH_EXCLUDE_FIELDS username
 ```
 
 
diff --git a/rootfs/etc/entrypoint.d/15-app.sh b/rootfs/etc/entrypoint.d/15-app.sh
index 4fe011a..bc21bb3 100755
--- a/rootfs/etc/entrypoint.d/15-app.sh
+++ b/rootfs/etc/entrypoint.d/15-app.sh
@@ -4,7 +4,7 @@ declare -x KANBOARD_DEBUG
 [[ -z "${KANBOARD_DEBUG}" ]] && KANBOARD_DEBUG="false"
 
 declare -x KANBOARD_PLUGIN_API_URL
-[[ -z "${KANBOARD_PLUGIN_API_URL}" ]] && KANBOARD_PLUGIN_API_URL="https://kanboard.net/plugins.json"
+[[ -z "${KANBOARD_PLUGIN_API_URL}" ]] && KANBOARD_PLUGIN_API_URL="https://kanboard.org/plugins.json"
 
 declare -x KANBOARD_PLUGIN_INSTALLER
 [[ -z "${KANBOARD_PLUGIN_INSTALLER}" ]] && KANBOARD_PLUGIN_INSTALLER="true"
@@ -183,6 +183,10 @@ declare -x KANBOARD_HTTP_VERIFY_SSL_CERTIFICATE
 declare -x KANBOARD_TOTP_ISSUER
 [[ -z "${KANBOARD_TOTP_ISSUER}" ]] && KANBOARD_TOTP_ISSUER="Kanboard"
 
+declare -x KANBOARD_EXTERNAL_AUTH_EXCLUDE_FIELDS
+[[ -z "${KANBOARD_EXTERNAL_AUTH_EXCLUDE_FIELDS}" ]] && KANBOARD_EXTERNAL_AUTH_EXCLUDE_FIELDS="username"
+
+
 declare -x KANBOARD_DB_DRIVER
 [[ -z "${KANBOARD_DB_DRIVER}" ]] && KANBOARD_DB_DRIVER="sqlite"
 
@@ -216,6 +220,9 @@ case "${KANBOARD_DB_DRIVER}" in
 
     declare -x KANBOARD_DB_SSL_CA
     [[ -z "${KANBOARD_DB_SSL_CA}" ]] && KANBOARD_DB_SSL_CA=""
+
+    declare -x KANBOARD_DB_VERIFY_SERVER_CERT
+    [[ -z "${KANBOARD_DB_VERIFY_SERVER_CERT}" ]] && KANBOARD_DB_VERIFY_SERVER_CERT=""
     ;;
 
   "postgres")
@@ -242,5 +249,8 @@ case "${KANBOARD_DB_DRIVER}" in
 
     declare -x KANBOARD_DB_SSL_CA
     [[ -z "${KANBOARD_DB_SSL_CA}" ]] && KANBOARD_DB_SSL_CA=""
+
+    declare -x KANBOARD_DB_VERIFY_SERVER_CERT
+    [[ -z "${KANBOARD_DB_VERIFY_SERVER_CERT}" ]] && KANBOARD_DB_VERIFY_SERVER_CERT=""
     ;;
 esac
diff --git a/rootfs/etc/templates/config.php.tmpl b/rootfs/etc/templates/config.php.tmpl
index 234592b..812da2e 100644
--- a/rootfs/etc/templates/config.php.tmpl
+++ b/rootfs/etc/templates/config.php.tmpl
@@ -28,6 +28,8 @@ define('DB_PORT', {{ envInt "DB_PORT" }});
 define('DB_SSL_KEY', {{ if eq (envString "DB_SSL_KEY") "" }}null{{ else }}'{{ envString "DB_SSL_KEY" }}'{{end}});
 define('DB_SSL_CERT', {{ if eq (envString "DB_SSL_CERT") "" }}null{{ else }}'{{ envString "DB_SSL_CERT" }}'{{end}});
 define('DB_SSL_CA', {{ if eq (envString "DB_SSL_CA") "" }}null{{ else }}'{{ envString "DB_SSL_CA" }}'{{end}});
+define('DB_VERIFY_SERVER_CERT', {{ if eq (envString "DB_VERIFY_SERVER_CERT") "" }}null{{ else }}{{ envBool "DB_VERIFY_SERVER_CERT" }}{{end}});
+define('DB_TIMEOUT', null);
 define('LDAP_AUTH', {{ envBool "LDAP_AUTH" }});
 define('LDAP_SERVER', '{{ envString "LDAP_SERVER" }}');
 define('LDAP_PORT', {{ envInt "LDAP_PORT" }});
@@ -75,3 +77,4 @@ define('HTTP_PROXY_USERNAME', '{{ envString "HTTP_PROXY_USERNAME" }}');
 define('HTTP_PROXY_PASSWORD', '{{ envString "HTTP_PROXY_PASSWORD" }}');
 define('HTTP_VERIFY_SSL_CERTIFICATE', {{ envBool "HTTP_VERIFY_SSL_CERTIFICATE" }});
 define('TOTP_ISSUER', '{{ envString "TOTP_ISSUER" }}');
+define('EXTERNAL_AUTH_EXCLUDE_FIELDS', '{{ envString "EXTERNAL_AUTH_EXCLUDE_FIELDS" }}');
